pallets/schemas/src/types.rs

```diff
@@ -11,7 +12,7 @@ use scale_info::TypeInfo;
 use sp_runtime::DispatchError;
 extern crate alloc;
 use alloc::{string::String, vec, vec::Vec};
-use frame_support::{pallet_prelude::DecodeWithMemTracking, traits::Len};
+use frame_support::traits::Len;
 
 /// Current storage version of the schemas pallet.
 pub const SCHEMA_STORAGE_VERSION: StorageVersion = StorageVersion::new(5);
```

Comment:
assuming this is already increase in previous PRs? or new version should 6?

Reply:
Correct; increased already in the main feature branch